firefox-ios/Client/Frontend/Browser/Toolbars/AddressToolbarContainer.swift

```diff
@@ -113,6 +143,7 @@ class AddressToolbarContainer: UIView,
                                                  profile: profile,
                                                  windowUUID: windowUUID)
         self.model = model
+        toolbarPosition = toolbarState.toolbarPosition
```

Comment:
```suggestion
        updateProgressBarPosition(toolbarState.toolbarPosition)
```

Reply:
Given the fact that the method updates the model, with no relation to the UI, I aimed to preserve the original intent and simply update the toolbar's position.